Tools/clinic/clinic.py

```diff
@@ -3005,6 +2996,14 @@ def parser_name(self):
 # note however that they will never be called with keyword-only parameters.
 legacy_converters: ConverterDict = {}
 
+# maps strings to callables.
+# these callables must be of the form:
+#   def foo(*, ...)
+# The callable may have any number of keyword-only parameters.
+# The callable must return a CConverter object.
+# The callable should not call builtins.print.
+return_converters: ConverterDict = {}
```

Comment:
This doesn't look right to me. The `converters` dict above is a mapping of `{str: Callable[..., CConverter}`. But this dictionary is a mapping of `{str: Callable[..., CReturnConverter}`.

Reply:
Good catch. I'll fix later.